pkg/config/config.go

```diff
@@ -172,6 +173,7 @@ type ServerConfig struct {
 	LogLevel string `toml:"log-level" json:"log-level"`
 
 	GcTTL int64  `toml:"gc-ttl" json:"gc-ttl"`
+	GcBLT int64  `toml:"gc-blt" json:"gc-blt"`
```

Comment:
Actually, is it better for this to be a global config item? It should be associated with the entire TiCDC cluster, but not any single node, nor any single changefeed. Just a thought.